### ISSUE TITLE
[key.dart] 1 << 32 not supported by Flutter Web

### ISSUE THREAD

**Issue by boeledi** [primes-network/eosdart_ecc#5]
First of all, many thanks for your library.

I am currently working with Flutter Web and it does not support the following statement "1 << 32"

Would you mind updating your "key.dart", line 114, the following way?

```
/// Generate the random EOS private key
  factory EOSPrivateKey.fromRandom() {
    //final int randomLimit = 1 << 32;
    final int randomLimit = 4294967296;
```

In advance, many thanks

**Comment by fzxu** [comment]
I am not familiar with Flutter Web. Is this issue still valid? @boeledi

**Comment by boeledi** [comment]
This is a limitation at the browser level, so yes it is.

**Comment by fzxu** [comment]
ok. this fixed in 0.4.1
